Makefile

```diff
@@ -15,6 +15,7 @@ NANOSERVER_VERSION := $(shell head -n 1 build/images/deps/nanoserver-version)
 WIN_BUILD_TAG      := $(shell echo $(GO_VERSION) $(CNI_BINARIES_VERSION) $(NANOSERVER_VERSION)|md5sum|head -c 10)
 GIT_HOOKS := $(shell find hack/git_client_side_hooks -type f -print)
 DOCKER_NETWORK     ?= default
+TRIVY_IMAGE        :=
```

Comment:
is this meant to be empty?

Reply:
Yes, since there are both ubuntu and ubi. I suppose to run this like Make trivy_scan TRIVY_IMAGE=XXXX

Reply:
then the target should at least print a user-friendly error if `TRIVY_IMAGE` is unset / empty. I also suggest renaming this variable to `TRIVY_TARGET_IMAGE`.